test/unit/api-client/api-client.submit.spec.ts

```diff
@@ -36,13 +36,38 @@ describe('api-client job submission', () => {
             expect(job).toEqual(jobDetails);
         });
 
+        it('submit job with media url with null options', async () => {
+            const mockHandler = ApiRequestHandler.mock.instances[0];
+            mockHandler.makeApiRequest.mockResolvedValue(jobDetails);
+            const options = {
```

Comment:
options isn't used in this test.

Reply:
delete it.

Reply:
Do you have tslint enabled on vscode? You should enable it and make sure that you fx any errors.

Also, we may want to add https://palantir.github.io/tslint/rules/no-unused-variable/ option to the tslint.json file in this repo.

Reply:
Apparently:
`no-unused-variable is deprecated. Since TypeScript 2.9. Please use the built-in compiler checks instead.`

I'm looking to see why the unused variable isn't being caught and how we can fix that.

Reply:
Actually it seems the entirety of `tslint` [has been deprecated](https://www.npmjs.com/package/tslint).

Reply:
Switch to ESLint proved to be a whole PR's worth of work, so will likely do in a separate issue sometime in the future.

During the attempted move to ESLint it came to my attention that we have several files especially the testing `.js` that do not currently get linted. Additionally while working with ESLint I realized there are a number of options that improve the code quality which we either don't have enabled or might not be working properly due to deprecation. This can all be addressed in a future ticket dedicated to this improvement.

Reply:
I let the mobile team know about the tslint deprecation. You should post about this int he #typescript channel.